src/common.cpp

```diff
@@ -32,19 +32,24 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
-// Example:
+// Right-handed shape broadcasting rules like in Numpy.
```

Comment:
Like doing long addition, move all the shapes to be right justified and compare columns.

Reply:
Okay. Makes sense now, but we probably need better term instead of right-handed shape broadcasting. 
Or you can add what you just described additionally i.e. " move all the shapes to be right justified and compare columns"

Reply:
suggested wording:  "// Right-handed shape broadcasting rules as in Numpy.  Last dimension must match last dimension, or be a broadcastable/fixed 1; next-to-last must match, etc."

Reply:
Yes, `multibroadcast` has a commutative shape function.

Reply:
Updated the comments to be clearer. Using the same explanation from numpy.